frontend/document/pdfdocument.lua

```diff
@@ -250,7 +251,7 @@ function PdfDocument:register(registry)
     registry:addProvider("xhtml", "application/xhtml+xml", self, 100)
     registry:addProvider("xml", "application/xml", self, 10)
     registry:addProvider("xps", "application/oxps", self, 100)
-    registry:addProvider("zip", "application/zip", self, 100)
+    registry:addProvider("zip", "application/zip", self, 10)
```

Comment:
Either this or crengine should be upped to `20`. Previously only MuPDF did zip; not sure why that one specifically.